src/pages/FirstRun.vue

```diff
@@ -20,20 +31,31 @@
 </template>
 
 <script lang="ts">
+import os from 'os';
+
 import { ipcRenderer } from 'electron';
+import EngineSelector from '@/components/EngineSelector.vue';
 import Vue from 'vue';
 
 import { Settings } from '@/config/settings';
 
 export default Vue.extend({
-  layout: 'dialog',
+  components: { EngineSelector },
+  layout:     'dialog',
   data() {
     return {
-      settings: { kubernetes: {} } as Settings,
-      versions: [] as string[],
+      settings:             { kubernetes: {} } as Settings,
+      versions:             [] as string[],
+      showContainerRuntime: false as boolean | unknown,
     };
   },
+  computed: {
+    hasContainerEnginePreferences() {
+      return !os.platform().startsWith('win');
+    },
+  },
   mounted() {
+    this.showContainerRuntime = process.env.showContainerRuntime as unknown;
```

Comment:
The `Kubernetes Settings` page now needs scrolling on a default screen. My first instinct would be to horizontally align the radio buttons to the right of the "Container Runtime" label, but this doesn't look possible with the `RadioGroup.vue` component because it puts the label and the button group inside an unidentified `div` (no `id`, no `class` attrs).  This does get ride of the scroll bar:
```


<style lang='scss'>
.radio-group.label {
  float: left;
  margin-left: 0px;
}
.radio-group {
  float: left;
  margin-left: 10px;
}
</style>
```

There's probably a better way of doing that.

Reply:
I introduced some changes to reduce the potential for vertical scrolling at our launch window size by reducing the label to `h4` and displaying options in row. Let me know how that works for you

![Screen Shot 2021-11-24 at 10 27 06 AM](https://user-images.githubusercontent.com/835961/143286908-11acd926-5e37-427a-802c-422f3be7c8a1.png)